reference/docs-conceptual/toc.yml

```diff
@@ -52,6 +52,24 @@
       href: learn/writing-portable-modules.md
     - name: PowerShell remoting
       items:
+        - name: Just Enough Administration (JEA)
```

Comment:
While you are fixing the validation issues, the indenting on the YML looks wrong.